politeiawww/backend.go

```diff
@@ -379,6 +384,42 @@ func (b *backend) emailUpdateUserKeyVerificationLink(email, publicKey, token str
 	return b.cfg.SMTP.Send(msg)
 }
 
+// emailUserLocked notifies the user its account has been locked and
+// emails the link with the reset password verification token
+// if the email server is set up.
+func (b *backend) emailUserLocked(email string) error {
+	if b.cfg.SMTP == nil {
+		return nil
```

Comment:
There is no way this is right. nil means everything is cool, this function literally is looking for the opposite. This function needs to return an error here that may be special handled but returning nil is not ok.

Reply:
That's how it is  being done at every method responsible to send email. 

It is the same to `emailNewUserVerificationLink`, `emailResetPasswordVerificationLink`, `emailUpdateUserKeyVerificationLink`. 

I believe we are not returning errors, because it is not necessary to have a mail server set to run `politeiawww`. Should we change and return an error in all these methods?